src/jwt/jwt-interceptor.js

```diff
@@ -0,0 +1,25 @@
+import axios from 'axios';
+import { router } from '../router/index.js';
+
+export default () => {
+    axios.interceptors.response.use( (response) => {
```

Comment:
Is this interceptor added to our axios instance?

Reply:
I would rather start with interceptor which adds JWT token to requests which auth is required instead of checking response for 401 code